doc/builders/fetchers.section.md

```diff
@@ -0,0 +1,62 @@
+# Fetchers {#chap-pkgs-fetchers}
```

Comment:
This file is a chapter so it needs to be named `fetchers.chapter.md`.

Reply:
I don't see the logic on it being a chapter: